File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/TestActivity.java
package com.zkteam.ui.components.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import com.zkteam.ui.components.R;

/**
 * Created by renxuelong on 2019-07-10
 */
public class TestActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);

        findViewById(R.id.btn_splash).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(TestActivity.this, SplashActivity.class));
            }
        });
    }
}

File: ZKUIComponents/src/main/java/com.zkteam.ui.components/activity/TestActivity.kt
package com.zkteam.ui.components.activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import com.zkteam.sdk.base.ZKBaseActivity
import com.zkteam.ui.components.R
import kotlinx.android.synthetic.main.activity_test.*

class TestActivity : ZKBaseActivity() {
    override fun getLayoutId(): Int {
        return R.layout.activity_test
    }

    override fun initData(bundle: Bundle?) {
        //function
    }

    override fun initListener() {
        btn_splash.setOnClickListener {
            startActivity(Intent(this, SplashActivity::class.java))
        }
    }

    override fun initViews(contentView: View) {
        //function
    }
}